server/s3_auth_client.h

```diff
@@ -358,6 +358,8 @@ class S3AuthClient {
  public:
   S3AuthClient(std::shared_ptr<RequestObject> req,
                bool skip_authorization = false);
+  bool set_get_method = false;
+  std::string clientabsoulte_uri = "";
```

Comment:
std::string constructor would set the clientabsolute_uri to empty string, no need to do it explicitly.